integration-tests/src/test/resources/compose/docker-compose.yml

```diff
@@ -7,13 +7,17 @@ services:
     - target: 8080
       mode: host
     environment:
-      - INDEX=noindex
+      - GLOBAL_db.driver=org.postgresql.Driver
+      - GLOBAL_db.url=jdbc:postgresql://postgresql:5432/alfresco
```

Comment:
do we need to add the driver and url now? Then we probably should fix the defaults in docker-alfresco

Reply:
the driver , and url not needed is not needed, will remove them.